external/vcm/vcm/derived_mapping.py

```diff
@@ -242,3 +247,36 @@ def pQ2(self):
         return self._mapper["pQ2"]
     except KeyError:
         return xr.zeros_like(self["pressure_thickness_of_atmospheric_layer"])
+
+
+@DerivedMapping.register(
+    "tendency_of_air_temperature_due_to_microphysics",
+    required_inputs=["air_temperature_input", "air_temperature_output"],
+)
+def tendency_of_air_temperature_due_to_microphysics(self):
+    end = self._mapper["air_temperature_output"]
+    begin = self._mapper["air_temperature_input"]
+    return (end - begin) / self._microphys_timestep_sec
+
+
+@DerivedMapping.register(
+    "tendency_of_specific_humidity_due_to_microphysics",
+    required_inputs=["specific_humidity_input", "specific_humidity_output"],
+)
+def tendency_of_specific_humidity_due_to_microphysics(self):
```

Comment:
Just an FYI, I'm not sure how the emulation branch plugs these into the prognostic run, but now the master version of the prognostic run code relies on hard coded mappings of tendency names to the state variable they update (https://github.com/ai2cm/fv3net/blob/a3fc28bf21eaa5ed9b2ca2a2a8cdeb4c4748e8f2/workflows/prognostic_c48_run/runtime/names.py#L22), so if the emulation models want to plug in that way in the future you just need to add the appropriate key/value pair for the prediction variables.

Reply:
Thanks for noting this.  Currently, the emulator plugs in through a different mechanism that doesn't interact with the wrapper state, but I'll keep this in mind for our other all-physics emulation work.